bin/elixir.ps1

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env pwsh
+
+$MinPowerShellVersion = [version]"7.2.0"
+
+if ($MinPowerShellVersion.CompareTo([version]$PSVersionTable.PSVersion) -eq 1) {
+  Write-Error "This script requires PowerShell version 7.2 or above. Running on $($PSVersionTable.PSVersion)"
+}
+
+$ELIXIR_VERSION = "1.17.0-dev"
```

Comment:
so this needs to be manually updated each release?

Reply:
Yes, we have instructions to update them. The reason is that people want to show the Elixir version on their terminal prompt, so we have a very quick way of computing it. I only suggest moving it to above `$MinPowerShellVersion = [version]"7.2.0"`, so it is easier to update.